### PR TITLE
docs: 缺少右括号

### PR DESCRIPTION
**Pull request by liuming-dev**
docs: 缺少右括号

### CODE REVIEW

**Comment by liuming-dev** [comment]
I signed it

File: content/zh/docs/concepts/overview/working-with-objects/labels.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ Similarly the comma separator acts as an _AND_ operator. So filtering resources
 The _set-based_ label selector is a general form of equality since `environment=production` is equivalent to `environment in (production)`; similarly for `!=` and `notin`.
 -->
 _基于集合_ 的标签选择算符是相等标签选择算符的一般形式，因为 `environment=production`
-等同于 `environment in（production`；`!=` 和 `notin` 也是类似的。
+等同于 `environment in（production)`；`!=` 和 `notin` 也是类似的。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/overview/working-with-objects/labels.md:242]: 请使用中文右括号，与左边保持一致。

Comment (tanjunchen) [reply content/zh/docs/concepts/overview/working-with-objects/labels.md:242]: Please sign the CLA, So we can review this.

**Review by tanjunchen — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**